src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,18 @@
+Microsoft.CodeAnalysis.ArtifactProducerAttribute
+Microsoft.CodeAnalysis.ArtifactProducerAttribute.ArtifactProducerAttribute() -> void
+Microsoft.CodeAnalysis.CommandLineArguments.GeneratedArtifactsOutputDirectory.get -> string?
+Microsoft.CodeAnalysis.Diagnostics.AnalysisContext.AnalysisContext(Microsoft.CodeAnalysis.Diagnostics.ArtifactContext? artifactContext) -> void
+Microsoft.CodeAnalysis.Diagnostics.AnalysisContext.TryGetArtifactContext(out Microsoft.CodeAnalysis.Diagnostics.ArtifactContext artifactContext) -> bool
+Microsoft.CodeAnalysis.Diagnostics.ArtifactContext
+Microsoft.CodeAnalysis.Diagnostics.ArtifactContext.ArtifactContext() -> void
+Microsoft.CodeAnalysis.Diagnostics.ArtifactContext.CreateArtifactStream(string! fileName) -> System.IO.Stream!
+Microsoft.CodeAnalysis.Diagnostics.ArtifactContext.WriteArtifact(string! fileName, Microsoft.CodeAnalysis.Text.SourceText! sourceText) -> void
+Microsoft.CodeAnalysis.Diagnostics.ArtifactContext.WriteArtifact(string! fileName, string! source) -> void
+Microsoft.CodeAnalysis.Diagnostics.ArtifactContext.WriteArtifact(string! fileName, System.Action<System.IO.Stream!>! writeStream) -> void
+Microsoft.CodeAnalysis.Diagnostics.ArtifactContext.WriteArtifact(string! fileName, System.Text.StringBuilder! builder) -> void
+Microsoft.CodeAnalysis.Diagnostics.IArtifactProducer
```

Comment:
This doesn't exist any more right?

Reply:
weird!  vs didn't give me any errors about this.  removed.